tools/ci_build/github/azure-pipelines/c-api-noopenmp-packaging-pipelines.yml

```diff
@@ -68,6 +68,11 @@ parameters:
     - 11.8
     - 12.2
 
+- name: QnnSdk
+  displayName: QNN SDK Version
+  type: string
+  default: 2.21.0.240401
```

Comment:
we're in process of updating default to 2.22 ? @adrianlizarraga fyi

Reply:
Thanks for the heads up. I'll update this to 2.22 in my branch once this PR is merged in.

Reply:
thanks